packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -1,51 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
-import { ThemeProvider as UiKitThemeProvider } from 'emotion-theming';
+import { css } from '@emotion/core';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
-import SpacingsInline from '@commercetools-uikit/spacings-inline';
-import Stamp from '@commercetools-uikit/stamp';
-import { Markdown, designSystem } from '@commercetools-docs/ui-kit';
+import { ReleaseNote, designSystem } from '@commercetools-docs/ui-kit';
```

Comment:
The `ReleaseNote` component does not belong in the uikit package. Move it to the theme components please.